winml/adapter/winml_adapter_c_api.cpp

```diff
@@ -99,10 +96,10 @@ static constexpr WinmlAdapterApi winml_adapter_api_1 = {
     &winmla::ReleaseModel
 };
 
-const WinmlAdapterApi* ORT_API_CALL OrtGetWinMLAdapter(_In_ const OrtApi* ort_api) NO_EXCEPTION {
-  if (OrtApis::GetApi(2) == ort_api) {
+const WinmlAdapterApi* ORT_API_CALL OrtGetWinMLAdapter(_In_ uint32_t version) NO_EXCEPTION {
+  if (version > 2) {
     return &winml_adapter_api_1;
```

Comment:
>1 [](http://example.com/codeflow?start=30&length=1)

Is this right? The previous code said `(2)`, but this check is `>2` rather than `>=2`?